public/app/features/alerting/unified/utils/matchers.ts

```diff
@@ -83,6 +89,14 @@ export function parsePromQLStyleMatcher(matcher: string): Matcher[] {
     }));
 }
 
+export function parsePromQLStyleMatcherLooseSafe(matcher: string): Matcher[] {
```

Comment:
Could we put JSDoc style comments explaining what the method intent is so its easier to see in IDEs? As it's not 100% clear what `loose safe` means out of context 😁

Reply:
Definitely needs a code comment here, especially since we have several similarly named functions with nuanced behaviour 👌